pages/index.js

```diff
@@ -70,7 +70,7 @@ class Index extends React.Component {
             <div className='content'>
                 <div className='banner'>
                     <div className='slogan'>
-                        <h1>Layer on top of Tensorflow for doing machine learning on encrypted data</h1>
+                        <h1>Encrypted Machine Learning in TensorFlow</h1>
```

Comment:
Believe this should be Encrypted Deep Learning -- which should also be in the website title.

Reply:
Not sure I agree with the title suggestion... usually this will get rendered like

![image](https://user-images.githubusercontent.com/1411145/60183073-eb31b680-97fb-11e9-8085-61af96794005.png)


vs 

![image](https://user-images.githubusercontent.com/1411145/60183102-f8e73c00-97fb-11e9-9784-a3dae446dc87.png)

Reply:
It's more about SEO then user viewing :)

Reply:
addressed in the seo pr